### DISCUSSION TITLE
Feat: Youtube Ambient Mode on lights off

### DISCUSSION THREAD

**Discussion by stabldev** [stabldev/coreproject-v3-ui#470] [Ideas]
![Screenshot from 2023-07-20 07-56-48](https://github.com/tokitou-san/CoreProject-V3-UI/assets/114811070/7d73d708-e82a-4ba1-af9f-ca61c174f6ca)

What about adding an ambient mode feat. on lights off
Uhh it would be a pain...

doing this in client may cause performance issues in low-end devices
and in server... check this
https://www.reddit.com/r/webdev/comments/yd7364/comment/itqln7s/

**Comment by baseplate-admin** [comment]
I need to research about how to use ffmpeg to generate meaningful frames. ( this will take  a while )

**Reply by stabldev** [reply]
Yea take your time... this is an optional feature
and I wont recommend to do this in client :)

reward: an ultimate fish you've never seen
